src/types/AddressData.d.ts

```diff
@@ -0,0 +1,13 @@
+export type AvailableHeating = 'collectif' | 'individuel' | undefined;
+export type AvailableStructure = 'Tertiaire' | 'Copropriété' | undefined;
+export type AddressDataType = {
+  address?: string;
+  company?: string;
+  geoAddress?: Record<string, any>;
+  eligibility?: boolean;
+  computEligibility?: boolean;
+  heatingType?: AvailableHeating;
+  network?: Record<string, any>;
+  structure?: AvailableStructure;
+  distance?: number;
```

Comment:
There is a lot of new field in this type, are you sure you're not mixing two types ?

Reply:
No, these are fine all the data that emerges from the eligibility/contact form and who is send from components to components inside an object.